.eslintrc.json

```diff
@@ -0,0 +1,26 @@
+{
+  "env": {
+    "browser": true,
+    "es2020": true,
```

Comment:
I'd set this to ES6 instead.  Where there are differences in what the linter recommends between the two, we'd probably want to go with the older style until we evaluated the impact of using the newer feature.

Reply:
Right, makes sense.